python/vosk/__init__.py

```diff
@@ -179,8 +173,8 @@ def SetPartialWords(self, enable_partial_words):
     def SetNLSML(self, enable_nlsml):
         _c.vosk_recognizer_set_nlsml(self._handle, 1 if enable_nlsml else 0)
 
-    def SetEpMode(self, mode):
-        _c.vosk_recognizer_set_ep_mode(self._handle, mode.value)
+    def SetSilence(self, length):
+        _c.vosk_recognizer_set_silence(self._handle, length)
```

Comment:
For me "set_silence" is not quite self-explaining. It might be set_silence_threshold probably. I think better name is required.

Reply:
"silence threshold" would make it sound like it's to do with energy/level of silence, whereas the min_trailing_silence values set the required silence *duration* (reference: https://github.com/kaldi-asr/kaldi/blob/master/src/online2/online-endpoint.h#L128). That said, the name probably could be better. Maybe set_required_pause or set_silence_duration?

Reply:
I've changed the name to set_pause, but can change it again if you'd prefer something else.